### PR TITLE
Add named volumes to binzx/otomi

### LINKED ISSUE #353: update otomi with workaround for docker run in linux
See last comments

### PR DESCRIPTION
**Pull request by svatwork**
I would like to continue building on this pr, the named volume addition will set the correct permissions for mounted files in the container, even on Linux hosts.

Closes #353.

### CODE REVIEW

**Review by Dunky13 — APPROVED** [review]
lgtm

File: binzx/otomi [hunk 1]
```diff
@@ -123,15 +123,36 @@ uname -a | grep -i darwin >/dev/null && helm_config="$HOME/Library/Preferences/h
...
+  source=$1
+  dest=$2
+  [ -z "$NAMED_VOLUME" ] && export NAMED_VOLUME='otomi-volume'
+  helper_container_name="otomi-volume-helper"
+
+  [ -z "$source" ] && echo "Error in function 'mount_with_named_volume': source path is empty" 1>&2
+
+  { # Create volume and copy over contents
+    docker container inspect $helper_container_name && docker rm $helper_container_name
+    docker container create --name $helper_container_name -v "$NAMED_VOLUME:$dest" hello-world &&
```
Comment (Morriz) [inline binzx/otomi:137]: "hello-world"?

And is this running on every command? Seems way too slow...

Comment (Morriz) [reply binzx/otomi:137]: maybe we can create it once and re-use?

**Review by Morriz — COMMENTED** [review]
would love to hear about perf impact

**Comment by svatwork** [comment]
Doesn't work, please revert. My bad.

**Comment by Morriz** [comment]
was it reverted already?

**Comment by Dunky13** [comment]
yes

**Pull request merged**